pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -201,7 +201,9 @@ func (c *ClusterConfig) ValidatePrivateCluster() error {
 		if c.VPC != nil && c.VPC.ID != "" && len(c.VPC.Subnets.Private) == 0 {
 			return errors.New("vpc.subnets.private must be specified in a fully-private cluster when a pre-existing VPC is supplied")
 		}
-		if additionalEndpoints := c.PrivateCluster.AdditionalEndpointServices; len(additionalEndpoints) > 0 {
+		if additionalEndpoints := c.PrivateCluster.AdditionalEndpointServices; len(additionalEndpoints) > 0 && c.PrivateCluster.SkipEndpointCreation {
+			return fmt.Errorf("additionalEndpoints may not be defined together with skipping endpoint creation")
```

Comment:
this error doesn't make sense to me, is `len(additionalEndpoints) > 0` mean that `additionalEndpoints` is defined? 🤔 
Can we reword this maybe?

Reply:
Is `additionalEndpoints may not be set together with skipping endpoint creation` better? :)

Reply:
hmm, how about `additionalEndpoints cannot be defined together with skipEndpointCreation set to true` ?

Reply:
sorry its friday, my brain is slow :D

Reply:
sounds good to me